hooks/post/30-upgrade.rb

```diff
@@ -80,6 +87,7 @@ def fail_and_exit(message)
     upgrade_step :import_puppet_modules, :long_running => true
     upgrade_step :import_subscriptions, :long_running => true
     upgrade_step :remove_elasticsearch
+    upgrade_set :remove_docker_v1_content, :long_running => true
```

Comment:
does this need to be done in hooks/pre/30-upgrade.rb to ensure that it happens prior to the rake db:migrate?